src/OrchardCore/OrchardCore.Roles.Core/CommonPermissions.cs

```diff
@@ -6,13 +6,16 @@ public static class CommonPermissions
 {
     public static readonly Permission ManageRoles = new("ManageRoles", "Managing Roles", isSecurityCritical: true);
 
+    [Obsolete("This Permission is no longer used and will be removed. Instead use OrchardCore.Users.CommonPermissions.AssignRoleToUsers.")]
     public static readonly Permission AssignRoles = new("AssignRoles", "Assign Roles", [ManageRoles], isSecurityCritical: true);
```

Comment:
Do I see correctly that this wasn't actually used anywhere even before? It had all the ceremony to generate dynamic permissions, but in the end, nothing authorized against it.

Reply:
Correct. It was used at some point in v1